test/unit/utils/ApplicationGenerator.scala

```diff
@@ -18,34 +18,31 @@ package unit.utils
 
 import java.util.UUID._
 
+import model.State._
 import model._
 import org.joda.time.DateTime
 import uk.gov.hmrc.time.DateTimeUtils
 
 object ApplicationGenerator {
 
+
+  def anApplicationWithHistory(applicationResponse: ApplicationResponse = anApplicationResponse(), stateHistories: Seq[StateHistory] = someStateHistories() ): ApplicationWithHistory =
+    ApplicationWithHistory(applicationResponse, stateHistories)
+
   def anApplicationResponse(dateTime: DateTime = DateTimeUtils.now): ApplicationResponse =
     ApplicationResponse(randomUUID(), "appName", "deployedTo", None, Set.empty, dateTime, Privileged(), ApplicationState(),
-      RateLimitTier.BRONZE, Some("termsUrl"), Some("privacyPolicyUrl"), None, None)
-
-  def anApplicationResponseWith(approval: ApprovedApplication): ApplicationResponse = anApplicationResponse().copy(approvedDetails = Some(approval))
+      RateLimitTier.BRONZE, Some("termsUrl"), Some("privacyPolicyUrl"), None)
 
   def anApplicationResponseWith(checkInformation: CheckInformation): ApplicationResponse = anApplicationResponse().copy(checkInformation = Some(checkInformation))
 
-  def anApprovedApplication(dateTime: DateTime = DateTimeUtils.now): ApprovedApplication =
-    ApprovedApplication(
-      ApplicationReviewDetails(id = "reviewId", name = "appName", description = "app description", rateLimitTier = None,
-        submission = SubmissionDetails("submitterName", "submitterEmail", dateTime), reviewContactName = Some("review contact name"),
-        reviewContactEmail = Some("review contact email"), reviewContactTelephone = Some("review contact telephone"), applicationDetails = None),
-      admins = Seq.empty,
-      approvedBy = "approved by",
-      approvedOn = dateTime,
-      verified = true
-    )
-
   def aCheckInformation(): CheckInformation =
     CheckInformation(contactDetails = Some(ContactDetails("contactFullName", "contactEmail", "contactTelephone")),
       confirmedName = true, providedPrivacyPolicyURL = true, providedTermsAndConditionsURL = true,
       applicationDetails = Some("application details"))
 
+  def someStateHistories() = Seq.empty
```

Comment:
I would not bother with this - seems a little superfluous when Seq.empty can just be used in the parameter list instead.

Also some sounds like it's returning an option when it's actually just a sequence :(

Reply:
Only in Scala land - usually some means more than one but i see what you mean